Formula/r/rbenv.rb

```diff
@@ -21,9 +21,8 @@ class Rbenv < Formula
   uses_from_macos "ruby" => :test
 
   def install
-    inreplace "libexec/rbenv" do |s|
-      s.gsub! ":/usr/local/etc/rbenv.d", ":#{HOMEBREW_PREFIX}/etc/rbenv.d\\0" if HOMEBREW_PREFIX.to_s != "/usr/local"
-    end
+    # This is needed only on non-`/usr/local` prefixes, but do it unconditionally for an :all bottle.
+    inreplace "libexec/rbenv", ":/usr/local/etc/rbenv.d", ":#{HOMEBREW_PREFIX}/etc/rbenv.d\\0"
```

Comment:
Probably needs to be
```suggestion
    inreplace "libexec/rbenv", ":/usr/local/etc/rbenv.d", ":#{HOMEBREW_PREFIX}/etc/rbenv.d"
```